scenes/act3.md

```diff
@@ -128,7 +127,7 @@ r: *Oh nooooo è pericoloso, non farlooooo.*
 publish("act3",["roofhunter",22]);
 ```
 
-r: Questo è il motivo per cui abbiamo bisogno di ssfiorare la morte! Spassatela! Carpe diem! Sniffa coca dal ^culo^ di un bel bocconcino, #YOLO!
+r: Ed ecco perché abbiamo bisogno di sfiorare la morte! Spassatela! Carpe diem! Sniffa coca dal ^culo^ di un bel bocconcino, #YOLO!
```

Comment:
"hooker" perché bocconcino e non prostituta?

Reply:
In effetti letteralmente la traduzione è quella. Però prostituta è femminile, bocconcino è neutro. Non sappiamo se il protagonista ha un interesse sessuale per il sesso opposto o meno.

Reply:
Hai ragione, Nicky ci ha detto di "tirare una moneta" per scegliere genere e orientamento sessuale. Come genere io ho scelto maschile e come orientamento "bi". 
Che ne pensi?

Reply:
LOL, sì, l'avevo letto! Se non fosse fattibile saremmo costretti a scegliere un genere e ti darei ragione in toto. 
La neutralità del protagonista, però, è il punto forte di un titolo che parla di ansia. Un qualcosa che ti colpisce duro e ti emozione se puoi immedesimarti nel protagonista. Usando il genere maschile rischiamo di "tagliar fuori" dall'immedesimazione tutti quelli che si identificano con il genere femminile (o comunque non binario). Non so se sei d'accordo. In caso se vuoi posso ricontrollare i capitoli già tradotti per vedere se è possibile adeguarli al neutro, proponendoti direttamente le modifiche.